roombapy/entry_points.py

```diff
@@ -1,3 +1,4 @@
+# mypy: ignore-errors
```

Comment:
This module is intentionally excluded from type checking. It will be replaced with `click`.